validation/src/main/scala/hmda/validation/rules/lar/validity/V570.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Result }
+import hmda.validation.rules.EditCheck
+
+object V570 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V570"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when(lar.lienStatus is equalTo(1)) {
+      lar.rateSpread is equalTo("NA") or {
+        try {
```

Comment:
According to @jmarin's [comment on my recent PR](https://github.com/cfpb/hmda-platform/pull/298#discussion_r61296170), using `try ... catch` isn't preferred in Scala. A more idiomatic way is to use `Try()` and `.getOrElse`. Maybe something like this?:
```scala
  override def apply(lar: LoanApplicationRegister): Result = {
    when(lar.lienStatus is equalTo(1)) {
      lar.rateSpread is equalTo("NA") or {
        rateSpreadInRange(lar.rateSpread)
          .getOrElse(Failure(s"can't parse ${lar.rateSpread} as decimal"))
      }
    }
  }

  private def rateSpreadInRange(input: String): Try[Result] = {
    Try(BigDecimal(input) is between(BigDecimal("1.50"), BigDecimal("99.99")))
  }
```

This is still new to me, though, so don't take this as law. :)